modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/utils/RestUtils.java

```diff
@@ -88,4 +94,56 @@
         license.put(SW360Attributes.LICENSE_TEXT, sw360License.getText());
         return getHttpEntity(license, header);
     }
+
+    private static Map<String, String> convertSW360ExternalIdsToMapOfStrings(SW360Release sw360Release) {
+        Map<String, String> externalIds = new HashMap<>(sw360Release.getCoordinates());
+
+        if(sw360Release.getDeclaredLicense() != null) {
+            externalIds.put(SW360Attributes.RELEASE_EXTERNAL_ID_DLICENSES, convertSetOfStringsToString(sw360Release.getDeclaredLicense()));
+        }
+        if(sw360Release.getObservedLicense() != null) {
+            externalIds.put(SW360Attributes.RELEASE_EXTERNAL_ID_OLICENSES, convertSetOfStringsToString(sw360Release.getObservedLicense()));
+        }
+        if(sw360Release.getReleaseTagUrl() != null) {
+            externalIds.put(SW360Attributes.RELEASE_EXTERNAL_ID_OREPO, sw360Release.getReleaseTagUrl());
+        }
+        if(sw360Release.getSoftwareHeritageUrl() != null) {
+            externalIds.put(SW360Attributes.RELEASE_EXTERNAL_ID_SWHREPO, sw360Release.getSoftwareHeritageUrl());
+        }
+        if(sw360Release.getHashes() != null) {
+            externalIds.putAll(convertSetOfStringsOfHashesToMapOfStrings(sw360Release.getHashes()));
+        }
+        if(sw360Release.getChangeStatus() != null) {
+            externalIds.put(SW360Attributes.RELEASE_EXTERNAL_ID_CHANGESTATUS, sw360Release.getChangeStatus());
+        }
+
+        return externalIds;
+    }
+
+    private static Map<String, String> convertSetOfStringsOfHashesToMapOfStrings(Set<String> stringSet) {
+        Map<String, String> setMap = new HashMap<>();
+        int i = 1;
+        Iterator<String> it = stringSet.iterator();
+        while(it.hasNext()){
+            setMap.put(SW360Attributes.RELEASE_EXTERNAL_ID_HASHES + i, it.next());
+            i++;
+        }
+        return setMap;
+    }
+
+    private static String convertSetOfStringsToString(Set<String> stringSet) {
+        if(stringSet.isEmpty()) {
+            return null;
+        }
+
+        StringBuilder stringBuilder = new StringBuilder();
```

Comment:
```suggestion
        return stringSet.stream().collect(Collectors.joining("; "));
```
(or something like that)